src/stores/AppStore.js

```diff
@@ -181,7 +181,7 @@ export default class AppStore extends Store {
 
         if (!isMac) {
           const mainWindow = remote.getCurrentWindow();
-          mainWindow.restore();
+          mainWindow.show();
```

Comment:
It is necessary for Windows to use `restore()`. I would suggest to have a separate handling for `isMac`, `isWindows` and `isLinux`.